samples/javascript/E3_GetIsClear/index.js

```diff
@@ -34,4 +34,4 @@ function getCurrentConditions(location) {
             resolve(body.current_observation);
         });
     });
-}
```

Comment:
you could use [request-promise-native](https://github.com/request/request-promise-native) (it looks like there are a [few flavors too](https://www.npmjs.com/package/request#promises--asyncawait)) to make your getCurrentConditions an async/await too!

Reply:
Ooh, nice!